src/nifgen/examples/nifgen_script.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+
+import argparse
+import nifgen
+import numpy as np
+from scipy import signal
+import sys
+import time
+
+
+def calculate_sinewave():
+    time = np.arange(start=0, stop=10, step=0.01)    # np.arange([start,]stop, [step,]dtype=None)
```

Comment:
Now you can remove the comment part.

Reply:
I like the comments because they show what defaults we are using. I can remove though if you don't think that is useful information.